types/src/tx_type/order_matching.rs

```diff
@@ -326,7 +334,7 @@ impl GetBytes for OrderMatching {
     }
 
     fn bytes_len(&self) -> usize {
-        SIGNED_ORDER_MATCHING_BIT_WIDTH / TX_TYPE_BIT_WIDTH
+        SIGNED_ORDER_MATCHING_BIT_WIDTH / 8
     }
```

Comment:
`SIGNED_ORDER_MATCHING_BIT_WIDTH` also need to update?

Reply:
I mean you add oracle_prices_hash in bytes, so the `SIGNED_ORDER_MATCHING_BIT_WIDTH` also should add the length of   `oracle_prices_hash`?

Reply:
Oh, it's not necessary, because oracle prices hash is included in the order bytes.
```rust
fn get_bytes(&self) -> Vec<u8> {
  let oracle_prices_hash = self.oracle_prices.rescue_hash();
  let mut orders_bytes = Vec::with_capacity(ORDERS_BYTES);
  orders_bytes.extend(self.maker.get_bytes());
  orders_bytes.extend(self.taker.get_bytes());
  orders_bytes.extend(oracle_prices_hash);
  // Todo: do not resize, sdk should be update
  orders_bytes.resize(ORDERS_BYTES, 0);
  let bytes_len = self.bytes_len();
  let mut out = Vec::with_capacity(bytes_len);
  out.push(Self::TX_TYPE);
  out.extend_from_slice(&self.account_id.to_be_bytes());
  out.extend_from_slice(&self.sub_account_id.to_be_bytes());
  out.extend(rescue_hash_orders(&orders_bytes));
  out.extend_from_slice(&(*self.fee_token as u16).to_be_bytes());
  out.extend_from_slice(&pack_fee_amount(&self.fee));
  out.extend_from_slice(&self.expect_base_amount.to_u128().unwrap().to_be_bytes());
  out.extend_from_slice(&self.expect_quote_amount.to_u128().unwrap().to_be_bytes());
  assert_eq!(out.len(), bytes_len);
  out
}
```